### PR TITLE
fix: pipeline

### PR DESCRIPTION
**Pull request by mfranzke**
reverted some parts out of https://github.com/db-ui/mono/pull/256

### CODE REVIEW

File: packages/components/src/components/divider/model.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import { type GlobalProps, type GlobalState } from '../../shared/model';
+import { GlobalProps, GlobalState } from '../../shared/model';
 
-export type DBDividerDefaultProps = {
+export interface DBDividerDefaultProps {
```
Comment (nmerget) [inline packages/components/src/components/divider/model.ts:3]: This could be a type, only the imports are bad

**Review by dkolba — COMMENTED** [review]
LGTM

**Review by nmerget — APPROVED** [review]

**Comment by mfranzke** [comment]
> > LGTM
> 
> @dkolba there's a special button to express your love, that even also makes the world go round (for this PR).

again, @nmerget was faster …

**Comment by mfranzke** [comment]
are you looking at the pipeline with the same amount of excitement as I do ?

**Pull request merged**